src/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -238,45 +238,45 @@ public with sharing class AFFL_AccRecordType_TEST {
     }
 
     @isTest
-    public static void addRecordTypeToAcc() {        
-        setup();
-       
+    private static void addRecordTypeToAcc() {        
+        AFFL_AccRecordType_TEST.setup();
```

Comment:
Curious to see why we are adding AFFL_AccRecordType_TEST here?

Reply:
This is from our style guide https://salesforce.quip.com/YksuAU6bZ6gE#fHJACAiihkO.